distr_test/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "distr_test"
+version = "0.5.0-alpha.1"
+authors = ["The Rand Project Developers"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/rust-random/rand"
+homepage = "https://rust-random.github.io/book"
+description = """
+Testing of distributions of rand_distr
+"""
+edition = "2021"
+rust-version = "1.61"
```

Comment:
Since this crate isn't meant to be published, it should have `publish = false` and doesn't need many of the other fields (compare `benches/Cargo.toml`).